doc/source/groups_of_atoms.rst

```diff
@@ -0,0 +1,117 @@
+.. -*- coding: utf-8 -*-
+.. _groups-of-atoms:
+
+Groups of atoms
+===============
+
+MDAnalysis has a hierarchy of :class:`~MDAnalysis.core.groups.Atom` containers that are used throughout the code.
+
+.. image:: images/classes.png
+
+First and foremost is the :class:`~MDAnalysis.core.groups.AtomGroup`. An :class:`~MDAnalysis.core.groups.AtomGroup` is the primary :class:`~MDAnalysis.core.groups.Atom` container; virtually everything can be accessed through it, as detailed `above <https://www.mdanalysis.org/UserGuide/data_structures.html#atomgroup>`_. This includes chemically meaningful groups of :class:`~MDAnalysis.core.groups.Atom`\ s such as a :code:`Residue` or a :code:`Segment`. 
+
+---------------------
+Residues and Segments
+---------------------
+
+A :class:`~MDAnalysis.core.groups.Residue` is composed of :class:`~MDAnalysis.core.groups.Atom`\ s, and a :class:`~MDAnalysis.core.groups.Segment` is composed of :class:`~MDAnalysis.core.groups.Residue`\ s.
```

Comment:
Do the plurals render nicely? (I have to remember the backslash trick, I used to reformulate as "Atom instances"...)

Reply:
As well as it can.

<img width="674" alt="Screenshot 2019-10-26 at 2 16 09 PM" src="https://user-images.githubusercontent.com/31115101/67613818-929c1580-f7fd-11e9-9bf1-405fc1a38f72.png">

You can also change the text with `Atoms <MDAnalysis.core.groups.Atom>`. Is this too misleading?

<img width="668" alt="Screenshot 2019-10-26 at 2 17 37 PM" src="https://user-images.githubusercontent.com/31115101/67613825-b5c6c500-f7fd-11e9-9780-c02f730cec40.png">

Or probably not every plural has to be a class link. (I will change those lingering :code:Residues etc. to classes!)

Reply:
I wouldn't change the text. We also singular/plural attributes. Better to use the formatting to make clear what the actual name is.